modules/objdetect/include/opencv2/objdetect/face.hpp

```diff
@@ -0,0 +1,105 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef _OPENCV_DNN_FACE_CORE_HPP_
+#define _OPENCV_DNN_FACE_CORE_HPP_
```

Comment:
Underscore prefixes are not allowed in public headers. Issue: #7288

Reply:
Fixed at https://github.com/opencv/opencv/pull/20422/commits/8e9278ec8c19ed45635897f2f31e33e82e25d432.